src/governor.cc

```diff
@@ -56,7 +56,7 @@ void readComplete(napi_env env, napi_status asyncStatus, void *data) {
   }
   REJECT_STATUS;
 
-  c->status = napi_create_external_buffer(env, c->readLen, c->readBuf, readFinalizer, (void*)(uint64_t)c->readLen, &result);
```

Comment:
would `NODE_API_NO_EXTERNAL_BUFFERS_ALLOWED` be helpful to define? I saw that here when [reading docs](https://nodejs.org/api/n-api.html#napi_create_external_buffer)